python/dgl/data/ppi.py

```diff
@@ -60,18 +66,20 @@ def __init__(self, mode='train', raw_dir=None, force_reload=False, verbose=False
                                          force_reload=force_reload,
                                          verbose=verbose)
 
-    def process(self, root_path):
-        graph_file = os.path.join(root_path, '{}_graph.json'.format(self.mode))
-        label_file = os.path.join(root_path, '{}_labels.npy'.format(self.mode))
-        feat_file = os.path.join(root_path, '{}_feats.npy'.format(self.mode))
-        graph_id_file = os.path.join(root_path, '{}_graph_id.npy'.format(self.mode))
+    def process(self):
+        graph_file = os.path.join(self.save_path, '{}_graph.json'.format(self.mode))
+        label_file = os.path.join(self.save_path, '{}_labels.npy'.format(self.mode))
+        feat_file = os.path.join(self.save_path, '{}_feats.npy'.format(self.mode))
+        graph_id_file = os.path.join(self.save_path, '{}_graph_id.npy'.format(self.mode))
 
         g_data = json.load(open(graph_file))
         self._labels = np.load(label_file)
         self._feats = np.load(feat_file)
-        self.graph = DGLGraph(nx.DiGraph(json_graph.node_link_graph(g_data)))
+        self._graph = DGLGraph(nx.DiGraph(json_graph.node_link_graph(g_data)))
```

Comment:
if `graph` is a property, why not keep using it?

Reply:
Moved it to @property

Reply:
I mean can we keep using `self.graph` instead of `self._graph`